source/usage-examples/command.txt

```diff
@@ -0,0 +1,31 @@
+=============
+Run A Command
+=============
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can run all :manual:`non CRUD </crud/>` database operations using the 
```

Comment:
"non CRUD" in the link feels off, since there should be a hyphen, and the link leads to CRUD, not a definition of "non-CRUD". I'd suggest rewording it to avoid the "non-" part.

Reply:
updated to "You can run all raw database operations, not including CRUD operations, using the command() method."